### PR TITLE
Polyhedron_demo: Display distance between points

### PR DESCRIPTION
**Pull request by maxGimeno**
This PR allows the user to display the distance between two points in the scene by pressing D and clicking on the scene. It will display a line between the two points, print their coordinates on their position and the distance in the middle of the segment. Press D alone to erase it.

screen

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Viewer.cpp [hunk 1]
```diff
@@ -192,59 +199,111 @@ void Viewer::initializeGL()
...
-      "gl_FragColor = color*light_amb + diffuse + specular; \n"
-      "} \n"
-      "\n"
-  };
-  QOpenGLShader *vertex_shader = new QOpenGLShader(QOpenGLShader::Vertex);
-  if(!vertex_shader->compileSourceCode(vertex_source))
-  {
-      std::cerr<<"Compiling vertex source FAILED"<<std::endl;
-  }
+          "#version 120 \n"
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Viewer.cpp]: any reason to change the indentation here?

**Comment by sloriot** [comment]
If you're really fast to select the points, the previous text is not erased
screenshot from 2016-05-30 18-02-38

**Comment by sloriot** [comment]
When changing the rotation center (hold right and double-click), the distance segment is displayed in bold.
screenshot from 2016-05-30 18-11-34
Actually I'd prefer it that way all the time as it is not easy to distinguish it from mesh edges

**Comment by maxGimeno** [comment]
The bugs have been fixed.

**Pull request merged**